pwa/static/themes/utrecht.css

```diff
@@ -203,4 +203,10 @@
   --web-app-alert-info-action-button-background-color: hsl(207 80% 35%);
   --web-app-alert-info-action-button-background-color-hover: hsl(207 80% 30%);
   --web-app-alert-info-icon-color: hsl(207 80% 35%);
+
+  /* 
+  * Utrecht component overrides
+  */
+  --web-app-button-secondary-action-background-color: inherit;
+  --web-app-button-secondary-action-color: inherit;
```

Comment:
_Feedback also goes for the `rotterdam.css` file_

I think we should specify that these tokens are meant _only_ for the button _link_ variant. This name suggests that its the `..-color` and `..-background-color` for the button itself which is (or might) not (be) true.

In the `component-overrides`, we're wrapping the overwrite in `.utrecht-button-link--secondary-action`, making it explicitly _only_ relevant in the `-button-link...`.

I think the simplest solution would be to rename these `--web-app` tokens to (e.g.): `--web-app-button-link-....`.

Reply:
In the case of the ButtonLink from Utrecht, I think using `web-app-button` actually makes sense, even for `button-link`: the `ButtonLink` component uses the tokens from `Button`, because they  are intended to look identical.

So I think the token names are probably fine the way they are now.